### DISCUSSION TITLE
Arch Linux package distribution

### DISCUSSION THREAD

**Discussion by danielweck** [edrlab/thorium-reader#1740] [Show and tell]
Currently user-contributed by a third-party not involved with the project. It seems that the config file downloads the official Debian build from GitHub:

https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=thorium-bin

**Comment by danielweck** [comment]
Note this out-of-date Snap distribution:
https://snapcraft.io/install/thorium-reader-snap/arch

**Comment by elcste** [comment]
I'd love an official Snap package!

Edit: This is semi-off-topic, but the Snap you linked to seems to be up-to-date now, and works well in my brief testing.

**Comment by panaC** [comment]
I'm closing the issue feel free to discuss it in discussion

**Comment by danielweck** [comment]
Related: https://github.com/edrlab/thorium-reader/discussions/1750

**Comment by elcste** [comment]
Related: https://github.com/edrlab/thorium-reader/discussions/1755 (branching Snap to it's own thread).
